e2e/plain-views/src/empty.php

```diff
@@ -0,0 +1,2 @@
+<?php
+
```

Comment:
please remove space

```suggestion
```

Reply:
done. leads to the same result though.

I guess the current empty.php does not fulfil the `$stmts[0] instanceof FileWithoutNamespace` check